code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -513,16 +513,16 @@
 		var/treatment_method
 		var/list/potential_methods = list()
 
-		if(C.getBruteLoss() >= heal_threshold)
+		if((C.getBruteLoss() >= heal_threshold) && C.getBruteLoss() != 0)
```

Comment:
```suggestion
		if(heal_threshold != 0 && C.getBruteLoss() >= heal_threshold)
```
Instead of calling that getter proc twice, check if the threshold, which is a source variable (cheaper to check), is different from zero.
This also is more intuitive in general, you want to check for the damage only if the heal threshold is relevant.

Reply:
Ah, that would stop the derelict medibot in particular from even _attempting_ to heal, I think. The if statements need to evaluate to true, so that it appends the desired healtypes to `potential_methods`. 
It'll then pick from the `potential_methods` list to treat from.

Since heal_threshold for the derelict medibot is 0, all the checks will fail and no treatment methods will be added.

...in hindsight this could do with a proper refactor. I'll give it a think.

Reply:
Aha, I see what you mean. The recommendation to change it to `>` seems like a better alternative.
What you commented, `Because it would change the behaviour of medbots. The ones with the default threshold of 10 will now ignore mobs with 10 damage, and only heal when the damage is 11 or higher`, is not a thing, because damage uses decimals. You'll just need `10.01` damage or so to be healed, the difference is very small.

Reply:
Ah, good to know! I was naive to think damage simply worked as integers.